### PR TITLE
Use real world data for values in spec

### PR DESCRIPTION
**Pull request by kxhitiz**

### CODE REVIEW

File: rspec/use-real-world-data.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Use real world data
+
+Let's try to use real world data or something that is more intuitive for sample data in our specs.
+It helps in improving clarity and expectation of the spec.
+
+
+## Bad
+
+````ruby
+  BlogPostCategory.new(name: "mac and cheese")
```
Comment (nguyele) [inline rspec/use-real-world-data.md]: 😆

**Comment by lewispb** [comment]
Thanks for submitting this @kxhitiz - I think "Real Estate Marketing" is more _realistic_ than "Mac & Cheese". Arguably "Mac & Cheese" is more delicious but that's besides the point 😄 

What I was trying to explain in my comments on your PR was that what is more useful than _realistic_ is _descriptive_ data in tests.

For example:

```
## Bad
alice = create(:user, name: "Alice")
alice.friends << create(:user, name: "Bob")
create(:user, name: "Mary")

visit friends_path(alice)
expect(page).to have_text("Bob")
expect(page).to_not have_text("Mary") # Who is Bob/Mary and how are they related again?

## Good
user = create(:user)
user.friends << create(:user, name: "Friend of User")
create(:user, name: "Not Friend of User")

visit friends_path(user)
expect(page).to have_text("Friend of User")
expect(page).to_not have_text("Not Friend of User")
```

What do you think, does that make sense?

File: rspec/use-real-world-data.md (hunk 1 above)
Comment (lewispb) [reply rspec/use-real-world-data.md]: mmm 😋

**Comment by kxhitiz** [comment]
@lewispb yeah. will add this in addition to above example.

File: rspec/use-real-world-data.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+````ruby
+  BlogPostCategory.new(name: "Real Estate Marketing")
+````
+
+## Use of descriptive data
+
+## Bad
+
+````ruby
+BlogPostCategory.new(name: "Real Estate Marketing")
```
Comment (lewispb) [inline rspec/use-real-world-data.md]:
```suggestion
```

**Pull request merged**